src/tests/multihost/sssd/testlib/common/helper_functions.py

```diff
@@ -0,0 +1,47 @@
+"""
+This script contains helper functions that can be used in test.
+"""
+
+import time
+from sssd.testlib.common.expect import pexpect_ssh
+
+ds_instance_name = 'example1'
+
+
+def clear_only_domain_log(multihost):
+    """
+    This function will clear domain logs
+    """
+    client = multihost.client[0]
+    log_ssd = f'/var/log/sssd/sssd_{ds_instance_name}.log'
+    client.run_command(f'> {log_ssd}')
+
+
+def find_logs(multihost, log_name, string_name):
+    """This function will find strings in a log file
+    log_name: Absolute path of log where the search will happen.
+    string_name: String to search in the log file.
+    """
+    log_str = multihost.client[0].get_file_contents(log_name).decode('utf-8')
+    assert string_name in log_str
```

Comment:
Add assert fail also,
like 'assert string_name in log_str, f'failed to find string in log_str'